src/filter-function.js

```diff
@@ -0,0 +1,24 @@
+function filterFunction(todoList, filterParam) {
+    
+    const lowerCaseFilter = filterParam.text.toLowerCase();
+
+    return todoList.filter(todo => {
+        const lowerCaseTaskList = todo.task.toLowerCase();
+        const tasktext = lowerCaseTaskList.includes(lowerCaseFilter);
+        const hasTask = !lowerCaseFilter || tasktext;
+        
+        let radioFilter = true;
```

Comment:
Heres a bit of logic that might help you fix the radio button stuff. 

`const filterStatus = filter.completed;
        let isCompleted = todoStatus.includes(filterStatus);

        if(filterStatus === 'all') {
            isCompleted = true;
        }  
        return hasItem && isCompleted;`